src/Compute/Compute/Extension/AzureDiskEncryption/DisableAzureDiskEncryption.cs

```diff
@@ -270,10 +273,20 @@ private AzureOperationResponse<VirtualMachine> UpdateVmEncryptionSettings()
                 Tags = vmParameters.Tags
             };
 
-            return this.ComputeClient.ComputeManagementClient.VirtualMachines.CreateOrUpdateWithHttpMessagesAsync(
-                this.ResourceGroupName,
-                vmParameters.Name,
-                parameters).GetAwaiter().GetResult();
+            if (NoWait.IsPresent)
```

Comment:
So, I am not sure if this is going to work - this second call may depend on the first having completed.  We should revert this one and ask the compute team about it.

Reply:
Done